### PR TITLE
Update PagedDoraWorkerTest.java

### PR DESCRIPTION
**Pull request by thu-david**
update Rename and exists tests for PagedDoraWorker.

### What changes are proposed in this pull request?

I have added two unit test for PagedDoraWorker.

### Why are the changes needed?

Please clarify why the changes are needed. For instance,
  1. One function for test Rename in PagedDoraWorker.
  2.One function for test Exists in PagedDoraWorker.

### Does this PR introduce any user facing changes?

No

### CODE REVIEW

File: dora/core/server/worker/src/test/java/alluxio/worker/dora/PagedDoraWorkerTest.java [hunk 1]
```diff
@@ -752,4 +754,35 @@ private void createDummyFile(File testFile, int length) throws Exception {
...
+    assertFalse(mWorker.exists(f.getAbsolutePath(), ExistsPOptions.getDefaultInstance()));
+    mWorker.delete(rootPath, DeletePOptions.getDefaultInstance());
+    assertFalse(mWorker.exists(rootPath, ExistsPOptions.getDefaultInstance()));
+  }
+
+  @Test
+  public void testRename() throws IOException, AccessControlException {
+    File srcFolder = mTestFolder.newFolder("root");
+    String rootPath = srcFolder.getAbsolutePath();
+    mWorker.rename(rootPath, rootPath + "2", RenamePOptions.getDefaultInstance());
```
Comment (elega) [inline dora/core/server/worker/src/test/java/alluxio/worker/dora/PagedDoraWorkerTest.java:777]: can you use PathUtils.concatPath() instead?

Comment (thu-david) [reply dora/core/server/worker/src/test/java/alluxio/worker/dora/PagedDoraWorkerTest.java:777]: Ok,I wil

**Review by elega — APPROVED** [review]
LGTM

**Comment by elega** [comment]
alluxio-bot, merge this please.

**Pull request merged**